x/slashing/src/params.rs

```diff
@@ -32,6 +32,19 @@ pub struct SlashingParams {
     pub slash_fraction_downtime: Decimal256,
 }
 
+impl SlashingParams {
+    pub fn min_signed_per_window_u32(&self) -> u32 {
+        // NOTE: RoundInt64 will never panic as minSignedPerWindow is less than 1.
+        self.min_signed_per_window
+            .checked_mul(Decimal256::from_atomics(self.signed_blocks_window as u64, 0).unwrap())
+            .unwrap()
+            .to_string()
+            .parse::<f64>()
```

Comment:
Do they use floating point types in cosmos?

Reply:
They don't. There was no easy way to get i64 from the Decimal256 and I used this conversion. You're right, I have to implement better solution here.

Reply:
Updated